src/index.html

```diff
@@ -1,13 +1,63 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="stylesheet" href="./styles/product-card.css">
+    <link rel="stylesheet" href="./styles/stars.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        src="./images/imac.jpeg"
+        alt="monitor"
+        class="card__img"
+      >
+      <p class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </p>
+      <p class="card__code">
+        Код товара: 195434
+      </p>
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__feedback">
+          Отзывов: 5
+        </p>
+      </div>
+      <div class="card__price">
+        <p class="card__cost">
+          Цена:
+        </p>
+        <p class="card__value">
+          69 999 грн
+        </p>
+      </div>
+      <div class="card__buy">
+        <a 
+          href="#"
+          class="card__link"
+          data-qa="hover"
+        >
+          <input 
+            value="Купить"
+            type="button"
+            class="card__button"
+          >
+        </a>
```

Comment:
Don’t need to use input inside link 

```suggestion
        <a 
          href="#"
          class="card__link"
          data-qa="hover"
        >
          Купить
        </a>
```